src/lib.rs

```diff
@@ -5,6 +5,11 @@
 //! You can then communicate with the device via this object.
 
 extern crate attohttpc;
+#[macro_use]
+extern crate log;
+extern crate bytes;
+
+extern crate lynx;
```

Comment:
`lynx` should be removed now, I've renamed it to `attohttpc` to avoid confusion with the browser.